### ISSUE TITLE
Cosmetic glitch with tips in network view

### ISSUE THREAD

**Issue by mhuyck** [greenelab/adage-server#102]
I found a cosmetic glitch while testing PR #99. If you click on an edge or node near the slider at the top, the tip that pops up is partially overlapped by the slider. Might be tricky to fix since the slider and network are separate components.

**Comment by mhuyck** [comment]
![screen shot 2017-03-23 at 10 44 42 am](https://cloud.githubusercontent.com/assets/1151166/24253535/f9663ec6-0fb6-11e7-8382-83fe730a1b0b.png)

**Comment by dongbohu** [comment]
Yes, I found this issue too. It will be ideal if the tips window can be moved by users, but as far as I know, the tips window that I am using doesn't allow users to do that. I will check the tips library API again.

**Comment by mhuyck** [comment]
Allowing a user to move the tip would only partially solve the problem. I think the ideal solution would be to set a boundary inside which the network (including any tips) must remain confined. That would keep the diagram from "leaking" into the surrounding page elements. As I said: not sure how tricky that would be to implement with the given tools.
